examples/flutter_firebaseauth/lib/home_page.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:flutter_firebaseauth/authentication/authentication.dart';
+
+class HomePage extends StatelessWidget {
+  final AuthenticationAuthenticated userState;
+
+  const HomePage({Key key, this.userState}) : super(key: key);
```

Comment:
since we're only showing the user's displayName, can we inject just the data we need instead of the entire AuthenticationState?

Reply:
Changed to inject the FirebaseUser from the state instead of the entire state.